core/src/main/java/org/apache/iceberg/SnapshotSummary.java

```diff
@@ -283,8 +292,13 @@ void addedFile(ContentFile<?> file) {
           this.addedRecords += file.recordCount();
           break;
         case POSITION_DELETES:
+          DeleteFile deleteFile = (DeleteFile) file;
+          if (ContentFileUtil.isDV(deleteFile)) {
+            this.addedDVs += 1;
+          } else {
+            this.addedPosDeleteFiles += 1;
+          }
           this.addedDeleteFiles += 1;
```

Comment:
I am treating `delete files` as a generic concept. We will continue to track each type of delete as a delete file and will count DVs, position delete files, and equality delete files separately. I think this is consistent and won't require updating our manifest counts to differentiate between DVs and delete files.